src/main/java/school/lemon/changerequest/java/generics/Container/Container.java

```diff
@@ -0,0 +1,74 @@
+package school.lemon.changerequest.java.generics.Container;
+
+
+public class Container<T> implements ContainerInterface {
```

Comment:
The thing is, that you should implement generic interface, and it should looks like: 
`Container<T> implements ContainerInterface<T>`
This will help you avoid casting to T.